src/DynamicRupture/FrictionLaws/FrictionSolverCommon.h

```diff
@@ -18,19 +18,19 @@
  */
 namespace seissol::dr::friction_law::common {
 
-template <size_t Start, size_t End, size_t Step>
+template <size_t StartT, size_t EndT, size_t StepT>
```

Comment:
It needed something to resolve the naming conflict with the parameters (which were now uppercased)—but it can also be something else than a `T`

Reply:
Could we name something like First, Last, Step in that case? Or is that too much of a deviation or unconventional? Adding a T in the end does not sound so nice, IMO. But if there is no other choice, this should be fine.

Reply:
Not really, no, there's not much of a convention (as of now); we can do that. Though it may be good to keep the template arguments similar to the constexpr names inside, i.e. a suffix seems/seemed like the most logical choice there to me, so I chose the `T`. (feel free to suggest something else there)